talaiot/src/test/kotlin/com/cdsap/talaiot/publisher/graphpublisher/KInfluxDBContainer.kt

```diff
@@ -0,0 +1,5 @@
+package com.cdsap.talaiot.publisher.graphpublisher
+
+import org.testcontainers.containers.InfluxDBContainer
```

Comment:
just a question, what is the benefit to use testcontainers?

Reply:
It's an EndToEnd test basically. You don't mock the InfluxDB, you really test against. It provides you a stronger guarantee in this case that it actually works. Mocking InfluxDB would also solve this but I tend to prefer also having E2E test just in case